source/styleEditor.js

```diff
@@ -1,3 +1,14 @@
 $(document).ready(function(){
```

Comment:
I like that you decided to write a function rather than stick the code inside the doc ready function! Since you did this you can simply write `$(document).ready(formListener());` rather than creating an anonymous function.